python/README.md

```diff
@@ -76,4 +76,8 @@ from this directory. If you only want to execute a particular test module, then
 python -m test.my_module_name
 ```
 
-where `my_module_name` is the name of the test module to be executed (the name of the file without the .py).
+where `my_module_name` is the name of the test module to be executed (the name of the file without the .py). A simple fuzzing test was also done and can be found on `fuzzing_test.py`. This test is not run along the unittest, to run it use:
+```
```

Comment:
CI should run tests that exist to ensure they do not break. Even if it doesn't do true fuzzing and only runs one iteration.

Reply:
SGTM! A "not real fuzzing" call to both function was added in a unittest class and included in the call to `python -m test` . I'm thinking about not changing the readme in order not to mislead about whether `python -m test` does do fuzzing, wdyt?